include/sound/sof.h

```diff
@@ -27,6 +27,9 @@ struct snd_sof_pdata {
 
 	struct device *dev;
 
+	/* indicate how many first bytes shouldn't be loaded into DSP memory. */
```

Comment:
Typo in commit message: "It makes possible to put extra information to host at the beginning..."

Reply:
changed